sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/UnusedPrivateMember.cs

```diff
@@ -421,11 +420,11 @@ private static AccessorDeclarationSyntax GetAccessorSyntax(IMethodSymbol methodS
             }
 
             var candidateSymbols = allNodes
-                .Select(n => new { Syntax = n.SyntaxNode, SemanticModel = n.SemanticModel, Symbol = n.SemanticModel.GetSymbolInfo(n.SyntaxNode) })
+                .Select(n => new { Syntax = n.SyntaxNode, n.SemanticModel, Symbol = n.SemanticModel.GetSymbolInfo(n.SyntaxNode) })

```

Comment:
_Syntax = n.SyntaxNode_ could be shortened too, couldn't it?

I think it would be better to be consistent and either omit all unnecessary explicit names, or explicitly name everything.

Reply:
You can't, it would rename `Syntax` into `SyntaxNode`.

Reply:
Why is that a problem? Because it's also a class name?

Reply:
I'll need to rename usages but except that there is no problem.